### DISCUSSION TITLE
续费2年专业版，当时已显示到2026年，但今天又显示成免费版了

### DISCUSSION THREAD

**Discussion by DwyaneSuarez** [getlantern/lantern-forum-cn#766] [Q&A]
邮箱为 489431992@qq.com

已发送邮件，希望尽快回复

**Comment by DwyaneSuarez** [comment]
![lantern付款](https://github.com/user-attachments/assets/f6da1ece-30c3-431a-a271-f126fd19cd42)

@Mike777Khoo 

麻烦解决一下

**Comment by DwyaneSuarez** [comment]
<img width="930" alt="image" src="https://github.com/user-attachments/assets/4b8dcb93-480f-4378-8525-f27e1bcd5dd5">

另外，登录界面会有这样的报错

**Comment by MaxMKMax** [comment]
@DwyaneSuarez 您好. 请问您的邮件发到哪一个Lantern的邮箱?

**Reply by DwyaneSuarez** [reply]
mike@getlantern.org

**Comment by MaxMKMax** [comment]
@DwyaneSuarez 对了, 您可以帮我个忙吗? 您在付费的选项里, 帮我查看有支付宝的选项吗? 我们在做测试. 想了解用户可以看到吗. 谢谢您!

**Reply by DwyaneSuarez** [reply]
看不到

**Reply by JinGuangyuan** [reply]
> @DwyaneSuarez 对了, 您可以帮我个忙吗? 您在付费的选项里, 帮我查看有支付宝的选项吗? 我们在做测试. 想了解用户可以看到吗. 谢谢您!

你好，我在win11版本的支付界面可以看到支付宝选项，并且于8月3日通过支付宝购买1年专业版，但目前显示为免费版且无法获取邮箱验证码登录，麻烦帮忙解决下，谢谢！邮箱为153922086@qq.com
![Uploading 8550c60a9327c03d2c2805e07662636.jpg…]()

**Reply by JinGuangyuan** [reply]
![8550c60a9327c03d2c2805e07662636](https://github.com/user-attachments/assets/43d3b67c-1761-4e69-85c7-e7cc7699fa52)

**Comment by DwyaneSuarez** [comment]
@Mike777Khoo 请问解决了吗，什么时候能正常使用

**Reply by MaxMKMax** [reply]
@DwyaneSuarez 
您好. 已经把问题提交上去了. 刚好负责人请病假. 过两天会给您弄好. 他一上线我马上帮您追问解决问题!

不好意思久等了.

**Reply by DwyaneSuarez** [reply]
还没解决吗，能发验证码过来吗？

**Comment by DwyaneSuarez** [comment]
@Mike777Khoo 
能发个验证码过来吗
